manager/integration/tests/test_uninstall.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import os
+import time
+
+from common import SETTING_DELETING_CONFIRMATION_FLAG
+from common import batch_v1_api, core_api, get_longhorn_api_client # NOQA
+from common import RETRY_COUNTS, RETRY_INTERVAL_LONG
+
+@pytest.mark.uninstall   # NOQA
+def test_uninstall(batch_v1_api, core_api): # NOQA
+
+    client = get_longhorn_api_client()
+    lh_raw = "https://raw.githubusercontent.com/longhorn/longhorn"
+
+    longhorn_version = os.getenv("LONGHORN_REPO_BRANCH")
```

Comment:
Why not just prepare the manifest path from outside? like https://github.com/longhorn/longhorn-tests/blob/f47c2879d5dff13311d71f9312226e8da14a1690/test_framework/scripts/longhorn-setup.sh#L18-L18. Try to follow the same pattern.